streams/src/test/java/org/apache/kafka/streams/state/internals/CachingKeyValueStoreTest.java

```diff
@@ -56,25 +57,25 @@
 
 public class CachingKeyValueStoreTest extends AbstractKeyValueStoreTest {
 
+    private final static String TOPIC = "topic";
+    private static final String CACHE_NAMESPACE = "0_0-store-name";
     private final int maxCacheSizeBytes = 150;
     private InternalMockProcessorContext context;
     private CachingKeyValueStore store;
-    private InMemoryKeyValueStore underlyingStore;
+    private KeyValueStore<Bytes, byte[]> keyValueStore;
```

Comment:
Not sure about this variable rename, since the `CachingKeyValueStore` is also a "keyValueStore". I.e., the important thing about the one we're declaring here is that it's the "underlying" one.